help/destinations/catalog/advertising/tradedesk-emails.md

```diff
@@ -0,0 +1,122 @@
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+|Email|Email addresses (clear text) |Select the Email target identity when your source identity is an Email namespace.|
+|Email_LC_SHA256|Email addresses need to be hashed using SHA256 and lowercased. Be sure to follow any [email normalization] (https://github.com/UnifiedID2/uid2docs/tree/main/api#email-address-normalization) rules required. You won't be able to change this setting later. |Select the Email_LC_SHA256 target identity when your source identity is an Email_LC_SHA256 namespace.|
+
+## Email hashing requirements (#hashing-requirements)
+
+You can hash email addresses before ingesting them into Adobe Experience Platform, or use raw email addresses in the Experience Platform, and have the Platform hash them on activation. 
+
+To learn about ingesting email addresses in Experience Platform, see the [batch ingestion overview] (https://experienceleague.adobe.com/docs/experience-platform/ingestion/batch/overview.html?lang=en). 
+
+If you select to hash the email addresses yourself, make sure to comply with the following requirements: 
+
+*  Remove leading and trailing spaces. 
+*  Convert all ASCII characters to lowercase. 
+*  In gmail.com email addresses, remove the following characters from the username part of the email address: 
+      *  The period (. (ASCII code 46)). For example, normalize jane.doe@gmail.com to janedoe@gmail.com. 
+      *  The plus sign (+ (ASCII code 43)) and all subsequent characters. For example, normalize janedoe+home@gmail.com to janedoe@gmail.com. 
+
+##Hash on Activation (#hash-on-activation)
+
+Data from unhashed namespaces is automatically hashed by Platform upon activation to a destination.  
+
+Attribute source data is not automatically hashed. When your source field contains unhashed attributes, check the Apply transformation option, to have Platform automatically hash the data on activation. The Apply transformation option is only displayed when you select attributes as source fields. It is not displayed when you choose namespaces. 
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Segment export]** | You are exporting all members of a segment (audience) with the identifiers (email or hashed email) used in the Trade Desk destination. |
+| Export frequency | **[!UICONTROL Daily Batch]** | As a profile is updated in Experience Platform based on segment evaluation, the profile (identities) are updated once a day downstream to the destination platform. Read more about [batch uploads](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-types.html?lang=en#file-based).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+## Authenticate to Destination (#authenticate)
```

Comment:
Minor. This is a subsection of the section above, so let's please use ### in the heading.

Reply:
Updated.